src/core/forms/forms.py

```diff
@@ -748,3 +749,70 @@ def is_confirmed(self):
             return super().is_confirmed()
         else:
             return True
+
+
+class EmailForm(forms.Form):
+    subject = forms.CharField(max_length=1000)
+    cc = TagitField(
+        required=False,
+        max_length=10000,
+    )
+    bcc = TagitField(
+        required=False,
+        max_length=10000,
+    )
+    body = forms.CharField(widget=SummernoteWidget)
+    attachments = MultipleFileField(required=False)
+
+    def clean_cc(self):
+        cc = self.cleaned_data['cc']
+        return self.email_sequence_cleaner("cc", cc)
+
+    def clean_bcc(self):
+        cc = self.cleaned_data['bcc']
+        return self.email_sequence_cleaner("bcc", cc)
+
+    def email_sequence_cleaner(self, field, email_seq):
+        if not email_seq or email_seq  == '':
+            return tuple()
+
+        for address in email_seq:
+            try:
+                validate_email(address)
+            except ValidationError:
+                self.add_error(field, 'Invalid email address ({}).'.format(address))
+
+        return email_seq
+
+    def as_dataclass(self):
+        return email.EmailData(**self.cleaned_data)
+
+
+class SettingEmailForm(EmailForm):
+    """ An Email form that populates initial data using Janeway email settings
+
+    During initialization, the email and subject settings are retrieved,
+    matching the given setting_name
+    :param setting_name: The name of the setting (Group must be email)
+    :param email_context: A dict of the context required to populate the email
+    :param request: The instance of this HttpRequest
+    :param journal: (Optional) an instance of journal.models.Journal
+    """
+    def __init__(self, *args, **kwargs):
+        setting_name = kwargs.pop("setting_name")
+        email_context = kwargs.pop("email_context", {})
+        subject_setting_name = "subject_" + setting_name
```

Comment:
This assumes that subject settings are always named exactly after their corresponding email settings. Have you checked whether that is a safe assumption?